common/src/measurement.rs

```diff
@@ -0,0 +1,27 @@
+/*++
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    measurement.rs
+
+Abstract:
+
+    Mailbox measurement related types.
+
+--*/
+use zerocopy::{AsBytes, FromBytes};
+
+/// Caliptra DPE Measurement    
+#[repr(C)]
+#[derive(AsBytes, FromBytes, Default, Debug)]
+pub struct StashMeasurement {
```

Comment:
See comment about adding an svn field. We can also do this in a follow-up PR if you'd prefer.

Reply:
Would propose adding a [u32; 12] `svn_context` field and a u32 `svn` field. The context field can represent the hash of the key that bound `svn` to the code being measured.

If the code being measured has no SVN, then `svn_context` and `svn` can be all zeroes.

Reply:
Opened https://github.com/chipsalliance/caliptra-sw/issues/759 to track. We can add in a follow-up PR.